.github/workflows/test-application.yaml

```diff
@@ -13,37 +13,28 @@ jobs:
         runs-on: ubuntu-latest
 
         env:
-            SYMFONY__DATABASE__DRIVER: pdo_mysql
-            SYMFONY__DATABASE__VERSION: 5.7
-            SYMFONY__DATABASE__HOST: 127.0.0.1
-            SYMFONY__DATABASE__PORT: 3306
-            SYMFONY__DATABASE__NAME: sulu_test
-            SYMFONY__DATABASE__USER: root
-            SYMFONY__DATABASE__PASSWORD: root
+            DATABASE_URL: 'mysql://root@127.0.0.1/sulu_redirect_test?serverVersion=5.7'
+            SYMFONY_DEPRECATIONS_HELPER: weak
```

Comment:
For consistency with the other bundles I would disable it in the `prefer-lowest` case